src/humanloop/decorators/helpers.py

```diff
@@ -0,0 +1,9 @@
+import inspect
+from typing import Any, Callable
+
+
+def args_to_inputs(func: Callable, args: tuple, kwargs: dict) -> dict[str, Any]:
```

Comment:
Why are we having to use a function like this?

Reply:
Required to properly fill in a Tool Log's `inputs` field

Reply:
Added some clarifying comments